hudi-common/src/main/java/org/apache/hudi/avro/AvroSchemaUtils.java

```diff
@@ -605,14 +605,16 @@ public static String createSchemaErrorString(String errorMessage, Schema writerS
    * @param nullable nullability of column type
    * @return a new schema with the nullabilities of the given columns updated
    */
-  public static Schema createSchemaWithNullabilityUpdate(
+  public static Schema forceNullableColumns(
           Schema schema, List<String> nullableUpdateCols, boolean nullable) {
```

Comment:
nullableUpdateCols -> columns. We can eliminate the flag `nullalbe` because it is always true.